recipes/x265/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set version = "2.9" %}
+
+package:
+  name: x265
+  version: {{ version }}
+
+source:
+  url: https://bitbucket.org/multicoreware/x265/downloads/x265_{{ version }}.tar.gz
+  md5: 693ee4ce7929a59735f441107310f0bb
+
+build:
+  skip: true  # [vc<15]
```

```suggestion
  skip: true  # [vc<14]
```

the build succeded only because there was nothing to build with `vc<15`... 😕